src/editor/overlay_widget.cpp

```diff
@@ -590,6 +613,21 @@ EditorOverlayWidget::move_object()
         new_pos -= pm->get_offset();
       }
     }
+
+    // TODO: Temporarily disabled during ongoing discussion
```

Comment:
What about this?

Reply:
Intended, we need to wait for user feedback before we take a decision about that. We'll adapt that feature based on what people tell us would be best.